sql/upgrade/sl3.0.sql

```diff
@@ -458,12 +465,21 @@ UPDATE :slschema.vendor SET entity_id = (SELECT id FROM entity WHERE 'V-' || ven
 
 UPDATE :slschema.customer SET entity_id = coalesce((SELECT min(id) FROM entity WHERE 'C-' || customernumber = control_code), entity_id);
 
+--TODO: Do we need that for MC?
+INSERT INTO defaults(setting_key,value)
+    SELECT 'curr',array_to_string(array_agg(curr),':')
+    FROM :slschema.curr;
```

Comment:
Yes, we need a currency setting in the defaults table, but not like this: the company's functional currency is in the defaults table after MC. The other currencies are in the currency table.